static/app/views/settings/account/notifications/notificationSettingsByProjects.tsx

```diff
@@ -48,7 +56,15 @@ class NotificationSettingsByProjects extends AsyncComponent<Props, State> {
   }
 
   getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
-    return [['projects', '/projects/']];
+    return [
+      [
+        'projects',
+        `/projects/`,
+        {
+          query: {organization_id: this.props.organizationId},
```

Comment:
```suggestion
          query: {organizationId: this.props.organizationId},
```

To go with the other change.